### ISSUE TITLE
Initialize Neural Network Model

### ISSUE THREAD

**Issue by EmmanB1** [atolliver/emoji-generator#9]
CNN that inputs an image (and emoji as the label) and provides an output image
Two ways of doing this:
Create your own model
Use a pretrained model (do research on how it works)
